src/main/java/io/cdap/plugin/salesforce/plugin/BaseSalesforceConfig.java

```diff
@@ -77,6 +80,9 @@ public class BaseSalesforceConfig extends ReferencePluginConfig {
   @Nullable
   private String loginUrl;
 
+  private final String fqnPrefix = "salesforce://";
```

Comment:
final variable names should be `UPPERCASE_SEPARATED_BY_UNDERSCORES`

Reply:
I had it like that at first, but checkstyle failed when building the module with `Name 'FQN_PREFIX' must match pattern '^[a-z][a-zA-Z0-9]*$'`. I ended up removing the constant as it is not really needed when I used String.format.

Reply:
Oh, you need to make this a static variable. Forgot to mention that.